### ISSUE TITLE
Can you publish/release the latest version (0.7.0) to NPM?

### ISSUE THREAD

**Issue by VAllens** [paazmaya/shuji#36]
### Expected behaviour

What it should have accomplished.

### Actual behaviour

How to reproduce with minimal working example.

### Versions and environment

 * Operating system:
 * Node.js:
 * npm:
 * shuji:

Thank you and have some :stew:.

**Comment by paazmaya** [comment]
There are no meaningful changes, see https://github.com/paazmaya/shuji/compare/v0.6.1...master

**Comment by VAllens** [comment]
I noticed the parameter `--preserve`, but it didn't seem to work in `v0.7.0`.

**Comment by VAllens** [comment]
`shuji` is a good tool , it brings me efficient automated work.
**Thank you for your great contribution.**

I have a good idea, I am converting *.js.map files to *.ts files, and then converting *.ts files to *.d.ts files.
I have used `shuji` to convert `*.map` files to `*.ts` files.
And i have a good idea, develop another automated tool, convert `*.ts` files to `*.d.ts` files.

I encountered a challenge, there are some `AMD` modules, which are output through `RESTFul API`. I don't have `*.ts` files, I am trying, **ignore these modules at build stage?**
